pythonit_toolkit/emails/templates.py

```diff
@@ -3,22 +3,18 @@
 
 class EmailTemplate(str, Enum):
     # Grants
-    GRANT_APPROVED_TICKET_ONLY = "grant-approved-ticket-only"
-    GRANT_APPROVED_TICKET_ACCOMMODATION = "grant-approved-ticket-accommodation"
-    GRANT_APPROVED_TICKET_TRAVEL = "grant-approved-ticket-travel"
-    GRANT_APPROVED_TICKET_TRAVEL_ACCOMMODATION = "grant-approved-ticket-travel-accommodation"
-    GRANT_REPLY_APPLICANT_NEED_MORE_INFO = "grant-reply-applicant-need-more-info"
-    GRANT_WAITING_LIST = "grant-waiting-list"
-    GRANT_WAITING_LIST_UPDATE = "grant-waiting-list-update"
-    GRANT_VOUCHER_CODE = "grant-voucher-code"
-    GRANT_REJECTED = "grant-rejected"
+    GRANT_APPROVED = "grants/approved"
```

Comment:
Does this work? I am not sure AWS supports `/` in the template name 🤔

Reply:
how can I test it? The only way is in production?

Reply:
I don't have enough knowledge about how SES works, we should upload the templates to SES service somehow?
It should be possible to organise the templates in directories, not sure if this is the right way to do it... 🤔